Xamarin.Forms.Core/RectangleTypeConverter.cs

```diff
@@ -12,7 +12,7 @@ public override object ConvertFromInvariantString(string value)
 			if (value != null)
 			{
 				string[] xywh = value.Split(',');
-				if (   xywh.Length == 4
+				if (xywh.Length == 4
```

Comment:
the reason for this extra spaces is to align the members of the logical AND, like this
```
if (   expr0
    && expr1)
```

which is more readable (for complex or long expressions) than
```
if (expr0
    && expr1)
```

or
```
if (expr0 &&
    expr1)
```